### DISCUSSION TITLE
Unsteady restart from a steady solution?

### DISCUSSION THREAD

**Discussion by ArneVoss** [su2code/SU2#1964] [Q&A] [answered]
Hello everyone, 

What is the best/correct way to perform an unsteady restart from a steady solution?

My current work around is to copy and rename (or link) the steady solution twice (e.g. restart_00000.dat and restart_00001.dat) and then to restart the unsteady solution with RESTART_ITER= 2. Because SU2 requires both the .dat and the .csv file, this involves four file operations.

By the way, to me it appears that the parameter READ_BINARY_RESTART= YES/NO has no effect. When should I use that one? (Extension of question #1801)

Thank you in advance for your answers!
Arne Voß

**Comment by pcarruscag** [comment, marked as answer]
That's the way to do it, no one has implemented reading the old coordinates from the binary solution file yet (so the csv is needed).
READ_BINARY_RESTART only controls the format used to restart the solution, not the geometry (which only works with csv).

**Reply by ArneVoss** [reply]
Pedro, thank you for the confirmation.
